wsi_superpixel_guided_labeling/web_client/views/body/ActiveLearningView.js

```diff
@@ -248,6 +248,20 @@ const ActiveLearningView = View.extend({
         this.currentAverageCertainty = sum / certaintyArray.length;
     },
 
+    getAgreeChoice(index, predictionPixelmapElement, labelPixelmapElement) {
+        const label = labelPixelmapElement.values[index];
+        const labelCategories = labelPixelmapElement.categories;
+        if (labelCategories[label].label === 'default') {
+            // Label is default, so no choice has been made
+            // Once we use the config file, we can use the default
+            // category specified there insted of the raw string
+            return undefined;
+        }
+        const predictionCategories = predictionPixelmapElement.categories;
+        const prediction = predictionPixelmapElement.values[index];
+        return predictionCategories[prediction].label === labelCategories[label].label ? 'Yes' : 'No';
+    },
```

Comment:
This function replaces simply comparing the pixelmap values of the prediction and label annotation. I've seen cases where, for example, the prediction annotation doesn't have the "default" category.